docs/source/techniques/ISISPowder-OSIRIS-v1.rst

```diff
@@ -47,20 +47,45 @@ Methods
 --------
 The following methods can be executed on a OSIRIS object:
 
-- :ref:`run_diffraction_focusing_osiris_isis-powder-diffraction-ref`
+- :ref:`focus_osiris_isis-powder-diffraction-ref`
+- :ref:`create_vanadium_osiris_isis-powder-diffraction-ref`
 
 For information on creating a OSIRIS object see:
 :ref:`creating_osiris_object_isis-powder-diffraction-ref`
 
-.. _run_diffraction_focusing_osiris_isis-powder-diffraction-ref:
+.. _create_vanadium_osiris_isis-powder-diffraction-ref:
 
-run_diffraction_focusing
+create_vanadium
+^^^^^^^^^^^^^^^
+The *create_vanadium* method allows a user to process a vanadium run.
+
+On OSIRIS the following parameters are required when executing *create_vanadium*:
+
+- :ref:`run_number_osiris_isis-powder-diffraction-ref`
+- :ref:`subtract_empty_can_osiris_isis-powder-diffraction-ref`
+
+Example
+=======
+
+..  code-block:: python
+
+  osiris_example.create_vanadium(run_number="119977", subtract_empty_can=False)
+
+.. _focus_osiris_isis-powder-diffraction-ref:
+
+focus
 ^^^^^^^^^^^^^^^^^^^^^^^^
-The *diffraction_focusing* method allows a user to process a series of runs into a
+The *focus* method allows a user to process a series of runs into a
 focused dSpace workspace. Whilst processing the runs the scripts can apply any corrections
 the user enables.
 
-On OSIRIS the following parameters are required when executing *create_vanadium*:
+The available corrections are:
+
+- empty container subtraction which can be enabled using :ref:`subtract_empty_can_osiris_isis-powder-diffraction-ref` parameter.
```

Comment:
I am being fussy but "which" should have a comma before it., alternatively you can use "that" and not have a comma. Same for the line below. 

```suggestion
- empty container subtraction, which can be enabled using :ref:`subtract_empty_can_osiris_isis-powder-diffraction-ref` parameter.
```

Reply:
Okay